### ISSUE TITLE
Elasticsearch Failures on SonarQube 6.6-RC1

### ISSUE THREAD

**Issue by fiveshotsofespresso** [racodond/sonar-css-plugin#66]
After starting up a fresh 6.6-RC1 SonarQube Server on the embedded H2 database and trying to install the sonar-css-plugin v4.10, these errors are thrown in the web.log file. The quality profile appears to register successfully (although this required having started up the server successfully at least once -- putting the plugin in the extensions/plugins folder and booting for the first time resulted in startup failing). 

```
2017.10.11 15:45:34 ERROR web[][o.s.s.es.BulkIndexer] index [rules], type [rule], id [css:validate-property-value], message [java.lang.IllegalArgumentException: Document contains at least one immense term in field="htmlDesc" (whose UTF8 encoding is longer than the max length 32766), all of which were skipped.  Please correct the analyzer to not produce such terms.  The prefix of the first immense term is: '[60, 112, 62, 10, 32, 32, 32, 32, 66, 114, 111, 119, 115, 101, 114, 115, 32, 105, 103, 110, 111, 114, 101, 32, 105, 110, 118, 97, 108, 105]...', original message: bytes can be at most 32766 in length; got 317492]
2017.10.11 15:45:34 ERROR web[][o.s.s.es.BulkIndexer] index [rules], type [rule], id [css:experimental-property-usage], message [java.lang.IllegalArgumentException: Document contains at least one immense term in field="htmlDesc" (whose UTF8 encoding is longer than the max length 32766), all of which were skipped.  Please correct the analyzer to not produce such terms.  The prefix of the first immense term is: '[60, 112, 62, 10, 32, 32, 32, 32, 69, 118, 101, 110, 32, 116, 104, 111, 117, 103, 104, 32, 118, 101, 110, 100, 111, 114, 45, 115, 112, 101]...', original message: bytes can be at most 32766 in length; got 58983]
2017.10.11 15:45:34 ERROR web[][o.s.s.es.BulkIndexer] index [rules], type [rule], id [css:known-properties], message [java.lang.IllegalArgumentException: Document contains at least one immense term in field="htmlDesc" (whose UTF8 encoding is longer than the max length 32766), all of which were skipped.  Please correct the analyzer to not produce such terms.  The prefix of the first immense term is: '[60, 112, 62, 10, 32, 32, 32, 32, 84, 104, 101, 32, 108, 105, 115, 116, 32, 111, 102, 32, 115, 117, 112, 112, 111, 114, 116, 101, 100, 32]...', original message: bytes can be at most 32766 in length; got 110492]
2017.10.11 15:45:34 ERROR web[][o.s.s.es.BulkIndexer] index [rules], type [rule], id [scss:known-properties], message [java.lang.IllegalArgumentException: Document contains at least one immense term in field="htmlDesc" (whose UTF8 encoding is longer than the max length 32766), all of which were skipped.  Please correct the analyzer to not produce such terms.  The prefix of the first immense term is: '[60, 112, 62, 10, 32, 32, 32, 32, 84, 104, 101, 32, 108, 105, 115, 116, 32, 111, 102, 32, 115, 117, 112, 112, 111, 114, 116, 101, 100, 32]...', original message: bytes can be at most 32766 in length; got 110492]
2017.10.11 15:45:34 ERROR web[][o.s.s.es.BulkIndexer] index [rules], type [rule], id [scss:validate-property-value], message [java.lang.IllegalArgumentException: Document contains at least one immense term in field="htmlDesc" (whose UTF8 encoding is longer than the max length 32766), all of which were skipped.  Please correct the analyzer to not produce such terms.  The prefix of the first immense term is: '[60, 112, 62, 10, 32, 32, 32, 32, 66, 114, 111, 119, 115, 101, 114, 115, 32, 105, 103, 110, 111, 114, 101, 32, 105, 110, 118, 97, 108, 105]...', original message: bytes can be at most 32766 in length; got 317492]
2017.10.11 15:45:34 ERROR web[][o.s.s.es.BulkIndexer] index [rules], type [rule], id [scss:experimental-property-usage], message [java.lang.IllegalArgumentException: Document contains at least one immense term in field="htmlDesc" (whose UTF8 encoding is longer than the max length 32766), all of which were skipped.  Please correct the analyzer to not produce such terms.  The prefix of the first immense term is: '[60, 112, 62, 10, 32, 32, 32, 32, 69, 118, 101, 110, 32, 116, 104, 111, 117, 103, 104, 32, 118, 101, 110, 100, 111, 114, 45, 115, 112, 101]...', original message: bytes can be at most 32766 in length; got 58983]
2017.10.11 15:45:34 ERROR web[][o.s.s.es.BulkIndexer] index [rules], type [rule], id [less:validate-property-value], message [java.lang.IllegalArgumentException: Document contains at least one immense term in field="htmlDesc" (whose UTF8 encoding is longer than the max length 32766), all of which were skipped.  Please correct the analyzer to not produce such terms.  The prefix of the first immense term is: '[60, 112, 62, 10, 32, 32, 32, 32, 66, 114, 111, 119, 115, 101, 114, 115, 32, 105, 103, 110, 111, 114, 101, 32, 105, 110, 118, 97, 108, 105]...', original message: bytes can be at most 32766 in length; got 317492]
2017.10.11 15:45:34 ERROR web[][o.s.s.es.BulkIndexer] index [rules], type [rule], id [less:experimental-property-usage], message [java.lang.IllegalArgumentException: Document contains at least one immense term in field="htmlDesc" (whose UTF8 encoding is longer than the max length 32766), all of which were skipped.  Please correct the analyzer to not produce such terms.  The prefix of the first immense term is: '[60, 112, 62, 10, 32, 32, 32, 32, 69, 118, 101, 110, 32, 116, 104, 111, 117, 103, 104, 32, 118, 101, 110, 100, 111, 114, 45, 115, 112, 101]...', original message: bytes can be at most 32766 in length; got 58983]
2017.10.11 15:45:34 ERROR web[][o.s.s.es.BulkIndexer] index [rules], type [rule], id [less:unknown-less-functions], message [java.lang.IllegalArgumentException: Document contains at least one immense term in field="htmlDesc" (whose UTF8 encoding is longer than the max length 32766), all of which were skipped.  Please correct the analyzer to not produce such terms.  The prefix of the first immense term is: '[60, 112, 62, 10, 32, 32, 32, 32, 84, 104, 101, 32, 108, 105, 115, 116, 32, 111, 102, 32, 115, 117, 112, 112, 111, 114, 116, 101, 100, 32]...', original message: bytes can be at most 32766 in length; got 37743]
2017.10.11 15:45:34 ERROR web[][o.s.s.es.BulkIndexer] index [rules], type [rule], id [less:known-properties], message [java.lang.IllegalArgumentException: Document contains at least one immense term in field="htmlDesc" (whose UTF8 encoding is longer than the max length 32766), all of which were skipped.  Please correct the analyzer to not produce such terms.  The prefix of the first immense term is: '[60, 112, 62, 10, 32, 32, 32, 32, 84, 104, 101, 32, 108, 105, 115, 116, 32, 111, 102, 32, 115, 117, 112, 112, 111, 114, 116, 101, 100, 32]...', original message: bytes can be at most 32766 in length; got 110492]
2017.10.11 15:45:34 ERROR web[][o.s.s.e.RecoveryIndexer] Elasticsearch recovery - too many failures [10/20 documents], waiting for next run
2017.10.11 15:45:34 INFO  web[][o.s.s.e.RecoveryIndexer] Elasticsearch recovery - 20 documents processed [10 failures] | time=939ms
```

**Comment by racodond** [comment]
Hi @fiveshotsofespresso,

Thanks a lot for the notification! It seems that an issue I raised a while ago is back: https://jira.sonarsource.com/browse/SONAR-6632

I saw your message on the mailing-list. Let's see if SonarSource will bother fix that issue.

Have a good day!

David

**Comment by bartfastiel** [comment]
I created ticket https://jira.sonarsource.com/browse/SONAR-9965 for this issue and I start investigations.

**Comment by fiveshotsofespresso** [comment]
I should note that the issue was resolved for 6.6, so I'm gonna go ahead and close this. :)
